### PR TITLE
Setting improvements

### PR DESCRIPTION
**Pull request by Holzhaus**

### CODE REVIEW

File: src/setting.rs [hunk 1]
```diff
@@ -69,10 +69,61 @@ pub struct Setting {
...
+            SettingData::MySetting(MySetting::default()),
+        )
+    }
+
+    /// Create a new object containing the default values of a `MYSETTING2.DAT` file.
+    #[must_use]
+    pub fn default_mysetting2() -> Self {
+        Self::default_with_brand_and_data(
+            "PIONEER".into(),
+            SettingData::MySetting(MySetting::default()),
```
Comment (Swiftb0y) [inline src/setting.rs:122]:
```suggestion
            SettingData::MySetting2(MySetting2::default()),
```

Comment (Holzhaus) [reply src/setting.rs:122]: good catch, thx

File: Cargo.toml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 name = "rekordcrate"
 version = "0.2.0"
 authors = ["Jan Holthuis <jan.holthuis@ruhr-uni-bochum.de>"]
-description = "Library for parsing Pioneer Rekordbox device exports"
+description = "Libratry for parsing Pioneer Rekordbox device exports"
```
Comment (Swiftb0y) [inline Cargo.toml:5]: accidentally committed typo

**Review by Swiftb0y — APPROVED** [review]
LGTM

**Pull request merged**